packages/test/snapshots/src/test/serialized.spec.ts

```diff
@@ -53,7 +53,7 @@ describe(`Container Serialization Backwards Compatibility`, () => {
 
             // Check for default data store
             const response = await container.request({ url: "/" });
-            assert.strictEqual(response.status, 200, "Component should exist!!");
+            assert.strictEqual(response.status, 200, `Component should exist!! ${response.value}`);
```

Comment:
I think having the response posted here can be more informative, but let me know if anyone sees a problem with this.

Reply:
Since this assert is in a test, we don't risk exposing any information to the logs by printing out the entire `response.value`